docs/source/howto/installation.rst

```diff
@@ -223,24 +223,344 @@ the configuration directory ``~/project_a/.aiida`` will be used.
 Tuning performance
 ==================
 
-`#4002`_
+AiiDA supports running hundreds of thousands of calculations and graphs with millions of nodes.
+However, optimal performance at that scale might require some tweaks to the AiiDA configuration to balance the CPU and disk load.
 
+Here are a few general tips that might improve the AiiDA performance:
+
+  1. :ref:`Move the Postgresql database<how-to:installation:more:move_postgresql>` to a fast disk (SSD), ideally on a large partition.
+
+  2. Use AiiDA's tools for making :ref:`efficient incremental backups<how-to:installation:backup:repository>` of the file repository.
+
+  3. Prevent your operating system from indexing the file repository.
+
+     .. dropdown:: Disabling ``locate`` on Linux
+
+        Many Linux distributions include the ``locate`` command to quickly find files and folders, and run a daily cron job ``updatedb.mlocate`` to create the corresponding index.
+        A large file repository can take a long time to index, up to the point where the hard drive is constantly indexing.
+
+        In order to exclude the repository folder from indexing, add its path to the ``PRUNEPATH`` variable in the ``/etc/updatedb.conf`` configuration file (use ``sudo``).
+
+
+  4. The verdi deamon can manage an arbitrary number of parallel workers; by default only one is activated. If ``verdi daemon status`` shows the daemon worker(s) constantly at high CPU usage, use ``verdi daemon incr X`` to add ``X`` daemon workers.
+     It is recommended that the number of workers does not exceed the number of CPU cores (or, if possible, that you use one or two cores less than your machine has, to avoid to degrade the PostgreSQL database performance).
+
+.. _how-to:installation:running-on-supercomputers:
+
+Running on supercomputers
```

Comment:
I'd recommend that we move this section a bit further down to after "Backing up your installation" given that this section is probably slightly less relevant for the average user in comparison to the other sections.

Reply:
Ok, will move